File: Zadanie 3/main.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Ile ocen wprowadzić? ");
        int n = int.Parse(Console.ReadLine());

        double suma = 0.0;

        for (int i = 0; i < n; i++)
        {
            Console.Write("Podaj ocenę {i + 1}: ");
            double ocena = double.Parse(Console.ReadLine());
            suma += ocena;
        }

        double srednia = suma / n;
        Console.WriteLine("Średnia ocen: " + srednia);

        if (srednia >= 3.0)
        {
            Console.WriteLine("Uczeń zaliczył przedmiot.");
        }
        else
        {
            Console.WriteLine("Uczeń nie zaliczył przedmiotu.");
        }
    }
}

File: Zadanie 3/main.py
n = int(input("Ile ocen chcesz wprowadzić? "))

suma = 0.0

for i in range(n):
    ocena = float(input("Podaj ocenę {i + 1}: "))
    suma += ocena

srednia = suma / n
print("Średnia ocen:", srednia)

if srednia >= 3.0:
    print("Uczeń zaliczył przedmiot.")
else:
    print("Uczeń nie zaliczył przedmiotu.")
